docs/changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+* **15 Feb 2019:**
+  * Changed the signature of the `spatial alpha cloud upload` and `spatial alpha cloud launch` CLI commands: All positional arguments were replaced by mandatory flags. For more information, please run the spatial CLI with the `--help` flag.
```

Comment:
```suggestion
  * Changed the signature of the `spatial alpha cloud upload` and `spatial alpha cloud launch` CLI commands: replaced all positional arguments with mandatory flags. For more information, please run the spatial CLI with the `--help` flag.
```

What does "run the spatial CLI with the --help flag" mean? Do you just need to run --help from within a SpatialOS project, or do you need to run one of the commands followed by --help?

Reply:
It's supposed to mean "run the commands with the --help flag, e.g. spatial alpha cloud upload --help"

Reply:
In that case I'd go for "please run the commands with the `--help` flag" 👍